.github/workflows/build_push.yml

```diff
@@ -0,0 +1,32 @@
+name: Build and Push Docker Images
+
+on:
+  pull_request:
```

Comment:
If you don't want to also copy the master.yml (the one that bumps release tags on every master merge), then I would change this block to read:
```
on:
    push:
````

I _think_ push is the right event

Reply:
thanks. done @EngHabu